src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -477,7 +475,7 @@ void CouplingSchemeConfiguration::addTransientLimitTags(
   } else {
     allowedMethods = {VALUE_FIXED};
   }
-  auto attrMethod = makeXMLAttribute(ATTR_METHOD, VALUE_FIXED).setOptions(allowedMethods);
+  auto attrMethod = makeXMLAttribute(ATTR_METHOD, VALUE_FIXED).setOptions(allowedMethods).setDocumentation("Is the time window size fixed or does the first participant choose it?");
```

Comment:
Should there be a question in the documentation?

Reply:
Good point! I reworded this doc string.